addons/website_forum/tests/test_forum_internals.py

```diff
@@ -145,3 +147,41 @@ def test_tags_usage_unused(self):
 
         self.assertEqual(self.base_forum.tag_most_used_ids, self.base_forum.tag_ids[:MOST_USED_TAGS_COUNT])
         self.assertEqual(self.base_forum.tag_unused_ids, self.env['forum.tag'])
+
+
+@tagged('forum_internals')
+class TestForumController(TestForumCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.forum2, cls.forum3 = cls.env['forum.forum'].create([{'name': f'TestForum{idx + 2}'} for idx in range(2)])
```

Comment:
As we add a test,  can we add a "other website" forum too? like forum4 would have another website so it is *not* shown even though we wrote a post in it?